templates/terraform/pre_update/app_engine_domain_mapping.erb

```diff
@@ -0,0 +1,28 @@
+<%- # the license inside this block applies to this file
+	# Copyright 2019 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+updateMask := []string{}
+if d.HasChange("ssl_settings") {
```

Comment:
Does this file mean that `update_mask_fields` doesn't quite work for this resource? If so, could you add a comment explaining why? (probably best to stick it here and in terraform.yaml)

Reply:
I was not aware that field existed - and it worked!  So I changed it to that :).  Thanks!